src/routes.ts

```diff
@@ -22,94 +21,90 @@ import GameStart from 'pages/GameStart';
 import GamePlay from 'pages/GamePlay';
 import GameOver from 'pages/GameOver';
 import EditPassword from 'pages/EditPassword';
-// const Home = loadable(() => import('pages/Home'));
-// const Login = loadable(() => import('pages/Login'));
-// const Register = loadable(() => import('pages/Register'));
-// const Profile = loadable(() => import('pages/Profile'));
-// const LeaderBoard = loadable(() => import('pages/LeaderBoard'));
-// const Forum = loadable(() => import('pages/Forum'));
-// const GameStart = loadable(() => import('pages/GameStart'));
-// const GamePlay = loadable(() => import('pages/GamePlay'));
-// const GameOver = loadable(() => import('pages/GameOver'));
-// const EditPassword = loadable(() => import('pages/EditPassword'));
-// import { Dispatch } from 'react';
-// import { match } from 'react-router';
 import { Action } from 'redux';
+import _identity from 'lodash/identity';
 
 export interface ReduxAction<T = any, P = any> extends Action {
   type: T;
   payload?: P;
 }
-// type RouterFetchDataArgs = {
-//   dispatch: Dispatch<ReduxAction>;
-//   match: match<{ slug: string }>;
-// };
 
 export default [
   {
     path: HOME_ROUTE,
     component: Home,
     needAuth: false,
```

Comment:
```suggestion
    needAuth: false,
```
баг. если пользователь авторизован то компонент с needAuth: false не подгрузится

Reply:
исправил ошибку с needAuth, некорректное условие отображения роутов написал